vendor/github.com/Cray-HPE/shasta-cfg/customizations.yaml

```diff
@@ -732,6 +737,8 @@ spec:
         ingresses:
           ingressgateway:
             issuers:
+              shasta-cmn: https://api.cmn.{{ network.dns.external }}/keycloak/realms/shasta
```

Comment:
We don't want these added back.   This is another change that needs to be done in the shasta-cfg repo.   Removing these two lines.

Reply:
Looks like CASMPET-5797.